app/lib/BaseEditorController.php

```diff
@@ -761,22 +785,10 @@ public function SummaryData($pa_options=null, $pb_ajax_load_displays=FALSE) {
 		    $vn_display_id = sizeof($va_tmp) > 0 ? array_shift(array_keys($va_tmp)) : 0;
 		}
 
-		// save where we are in session, for "Save and return" button
-		$va_save_and_return = Session::getVar('save_and_return_locations');
-		if(!is_array($va_save_and_return)) { $va_save_and_return = array(); }
-
-		$va_save = array(
-			'table' => $t_subject->tableName(),
-			'key' => $vn_subject_id,
-			'url_path' => $this->getRequest()->getFullUrlPath()
-		);
-
-		Session::setVar('save_and_return_locations', caPushToStack($va_save, $va_save_and_return, __CA_SAVE_AND_RETURN_STACK_SIZE__));
-
 		$this->view->setVar('bundle_displays', $va_displays);
 		$this->view->setVar('t_display', $t_display);
 
-		if (!$pb_ajax_load_displays || $this->request->isAjax()) {
+		if ($pb_load_placement_info) {
```

Comment:
If we invert the logic here and return on false, the code is a lot easier to compare to what is upstream (As it no longer needs to be indented, so git can match them up). 
Bit weird though.